docs/returning.md

```diff
@@ -0,0 +1,12 @@
+# Returning
```

Comment:
IMO we don't need these docs at all

Reply:
I agree, also this file as is doesn't appear in the docs anyway, just remove this file